docs/code-of-conduct-shortform-meetups.rst

```diff
@@ -1,18 +1,14 @@
-Code of Conduct
-===============
+Code of Conduct (Meetups)
+=========================
 
-Write the Docs is a global community of documentarians who share information, discuss ideas, and work together to improve the art and science of documentation. Everybody who participates in our community in one way or another is required to conform to our `Code of Conduct <https://www.writethedocs.org/code-of-conduct/>`_ (CoC). This includes online communications, meetup and conference attendees, speakers, sponsors, founders, moderators, organisers and volunteers.
+Write the Docs is dedicated to providing a safe, inclusive, welcoming, and harassment-free space and experience for all community participants, regardless of gender identity and expression, sexual orientation, disability, physical appearance, socioeconomic status, body size, ethnicity, nationality, level of experience, age, religion (or lack thereof), or other identity markers. 
 
-Our Code of Conduct applies to all spaces and events relating to this meetup. We strongly recommend you to read the full Code of Conduct, but for your convenience, our key principles are:
+Our Code of Conduct exists because of that dedication, and we do not tolerate harassment in any form. See our reporting guidelines `here <../code-of-conduct/#reporting-and-contact-information>`_. Our full Code of Conduct can be found at `this link <../code-of-conduct>`_.
 
-* **Be friendly and patient.**
+If you're a meetup leader, when possible, include the following excerpt in your opening remarks:
 
-* **Be welcoming.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+**"We are dedicated to providing a safe, inclusive, welcoming, and harassment-free space and experience for all community participants."**
```

Comment:
Can we add a suggestion for organisers to make the people that can be spoken to in case of an incident visible? As in, have them stand up, or visible near the front and pointed out? And/or mention that people can email conduct@writethedocs.org ?

So that "we are dedicated to provide a good experience" is followed up by "and here's what you can do if you don't have a good experience".

Reply:
While I'd rather avoid two CoCs for meetups, this makes me wonder. While your suggestion IMO is good at larger meetups, I don't think this is practical at the "coffee klatch" style meetup, where there are no slides or even a "front" of the meetup.

Reply:
I do include the conduct@writethedocs.org address elsewhere.

Reply:
It seems to me that this sort of thing is best left up to a meetup organizer. As Mike points out, there's a wide range of meetup forms in practice. The discussion up to this point has also been about the responsibilities of a meetup organizer. Surely we can trust organizers to figure out how best to promote the CoC? And the reporting of incidents?